lib/jsonite/lets_proxy.rb

```diff
@@ -0,0 +1,23 @@
+class Jsonite
+  class LetsProxy < BasicObject
+
+    undef_method :==, :!, :!=
+
+    def initialize object, context, lets = {}
+      @__object, @__context, @__lets = object, context, lets
```

Comment:
The `_*` variables are shadowed local variables, while these are private instance variables in the fashion of `Delegate`. In fact the usual pattern is `@__variable__`, which might be preferable. Thoughts?